rest/data/project.go

```diff
@@ -50,6 +53,45 @@ func FindProjectById(id string, includeRepo bool, includeProjectConfig bool) (*m
 	return p, nil
 }
 
+// RequestS3Creds creates a JIRA ticket that requests build to create an AWS user for the specified project.
```

Comment:
should remove build from this since we aren't hardcoding it; also probably should say s3 creds haha